openadapt/models_tuning/fine_tune_models.py

```diff
@@ -0,0 +1,207 @@
+from copy import deepcopy
+from pprint import pformat
+import math
+
+from loguru import logger
+import fire
+import transformers as tf
+
+from openadapt import crud, models, utils
+
+LOG_LEVEL = "INFO"
+MAX_SCREEN_SIZE = (1920, 1080)
```

Comment:
Can get this from @openadapt.models.recording https://github.com/OpenAdaptAI/OpenAdapt/blob/main/openadapt/models.py#L29 ?

Reply:
Thanks. I will take that into account.